src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -46,110 +59,80 @@ const propTypes = {
-    source: PropTypes.oneOf(['Chronos', 'email', 'ios', 'android', 'web', 'email', '']),
+    /** Should the comment have the appearance of being grouped with the previous comment? */
+    displayAsGroup: PropTypes.bool.isRequired,
 
-    /** Should this fragment be contained in a single line? */
-    isSingleLine: PropTypes.bool,
+    // Properties required for a TEXT fragment
 
-    // Additional styles to add after local styles
-    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+    /** User's accountID */
+    accountID: PropTypes.number.isRequired,
 
     /** The accountID of the copilot who took this action on behalf of the user */
     delegateAccountID: PropTypes.number,
 
-    /** icon */
-    actorIcon: avatarPropTypes,
-
-    ...windowDimensionsPropTypes,
+    /** Should this fragment be contained in a single line? */
+    isSingleLine: PropTypes.bool,
 
-    /** localization props */
-    ...withLocalizePropTypes,
+    /** The actor's icon */
+    actorIcon: avatarPropTypes,
 };
 
 const defaultProps = {
-    isAttachment: false,
-    attachmentInfo: {
+    source: '',
+    pendingAction: undefined,
+    isUploadedAttachment: false,
+    uploadedAttachmentInfo: {
         name: '',
         size: 0,
         type: '',
         source: '',
     },
-    loading: false,
-    isSingleLine: false,
-    source: '',
-    style: [],
     delegateAccountID: 0,
+    isSingleLine: false,
     actorIcon: {},
 };
 
 function ReportActionItemFragment(props) {
-    switch (props.fragment.type) {
-        case 'COMMENT': {
-            // If this is an attachment placeholder, return the placeholder component
-            if (props.isAttachment && props.loading) {
-                return Str.isImage(props.attachmentInfo.name) ? (
-                    <RenderHTML html={`<comment><img src="${props.attachmentInfo.source}" data-expensify-preview-modal-disabled="true"/></comment>`} />
-                ) : (
-                    <View style={[styles.chatItemAttachmentPlaceholder]}>
-                        <ActivityIndicator
-                            size="large"
-                            color={themeColors.textSupporting}
-                            style={[styles.flex1]}
-                        />
-                    </View>
-                );
-            }
-            const {html, text} = props.fragment;
+    const fragment = props.fragment;
 
+    switch (fragment.type) {
+        case 'COMMENT': {
             // Threaded messages display "[Deleted message]" instead of being hidden altogether.
             // While offline we display the previous message with a strikethrough style. Once online we want to
             // immediately display "[Deleted message]" while the delete action is pending.
 
-            if ((!props.network.isOffline && props.hasCommentThread && props.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) || props.fragment.isDeletedParentAction) {
```

Comment:
Here is the part that rendered "[Deleted message]" bit; but I guess it should actually have precedence over everything else, including the `if (props.isAttachment && props.loading)` attachment path?